Gopkg.toml

```diff
@@ -12,6 +12,8 @@ required = [
   "github.com/knative/test-infra/scripts",
   "github.com/knative/test-infra/tools/dep-collector",
   "sigs.k8s.io/controller-tools/pkg/crd/generator",
+  # For load testing.
+  "github.com/rakyll/hey"
```

Comment:
Do we need to add this dep, is it being used anywhere here?

Reply:
PLZ check the issue #57.